core/templates/dev/head/forms/formBuilder.js

```diff
@@ -559,6 +561,9 @@ oppia.factory('rteHelperService', [
           RTE_COMPONENT_SPECS[componentId].customization_arg_specs),
         name: RTE_COMPONENT_SPECS[componentId].frontend_name,
         iconDataUrl: RTE_COMPONENT_SPECS[componentId].icon_data_url,
```

Comment:
The iconDataUrl is used for the toolbar icon now, so I'm not really sure how to refer to that icon file with just previewUrlTemplate.

Right now the components are all inline (oops left that out), but I'll add CSS fixes to address that.